vm-setup/roles/v1aX_integration_test/tasks/move_back.yml

```diff
@@ -16,7 +16,10 @@
     when: EPHEMERAL_CLUSTER == "kind"
 
   - name: Install Ironic in Source cluster (Ephemeral Cluster is minikube)
-    shell: "kustomize build /home/$USER/go/src/github.com/metal3-io/baremetal-operator/ironic-deployment/keepalived/ | kubectl apply -n {{ NAMESPACE }} -f -"
+    shell: "{{ BMOPATH }}/tools/deploy.sh {{ DEPLOY_BMO }} {{ DEPLOY_IRONIC }} {{ DEPLOY_TLS }} {{ DEPLOY_BASIC_AUTH }} {{ DEPLOY_KEEPALIVED }}"
```

Comment:
Here DEPLOY_BMO should be "false", since the previous deployment should still be running, we only removed Ironic. and DEPLOY_IRONIC should be "true"

Reply:
Addressed in #516